tests/openedx_tagging/core/tagging/test_views.py

```diff
@@ -1190,3 +1190,492 @@ def test_next_children(self):
         assert data.get("count") == self.children_tags_count[0]
         assert data.get("num_pages") == 2
         assert data.get("current_page") == 2
+
+    def test_create_tag_in_taxonomy_while_loggedout(self):
+        new_tag_value = "New Tag"
+
+        create_data = {
+            "tag": new_tag_value
+        }
+
+        response = self.client.post(
+            self.small_taxonomy_url, create_data, format="json"
+        )
+
+        assert response.status_code == status.HTTP_401_UNAUTHORIZED
+
+    def test_create_tag_in_taxonomy(self):
+        self.client.force_authenticate(user=self.user)
```

Comment:
Once you've fixed the `TaxonomyTagsView.permission_classes`, you'll need to change your authentication for these tests to `self.staff`. But it's worth leaving in a few checks to ensure that `self.user` is not allowed to create or update tags.